ibm/service/power/resource_ibm_pi_network.go

```diff
@@ -75,10 +75,25 @@ func ResourceIBMPINetwork() *schema.Resource {
 				Description: "PI network gateway",
 			},
 			helpers.PINetworkJumbo: {
-				Type:        schema.TypeBool,
+				Type:         schema.TypeBool,
+				Optional:     true,
+				Computed:     true,
+				Deprecated:   "deprecated use pi_network_mtu instead",
+				ExactlyOneOf: []string{helpers.PINetworkMtu, helpers.PINetworkJumbo},
+				Description:  "PI network enable MTU Jumbo option",
+			},
+			helpers.PINetworkMtu: {
+				Type:         schema.TypeInt,
+				Optional:     true,
+				Computed:     true,
+				ExactlyOneOf: []string{helpers.PINetworkMtu, helpers.PINetworkJumbo},
+				Description:  "PI Maximum Transmission Unit",
+			},
+			helpers.PINetworkAccessConfig: {
```

Comment:
Please add validation for all the possible values. https://github.com/IBM-Cloud/power-go-client/blob/master/power/models/access_config.go#L39-L51
Example:
```
helpers.PINetworkType: {
    Type:         schema.TypeString,
    Required:     true,
    ValidateFunc: validate.ValidateAllowedStringValues([]string{"vlan", "pub-vlan"}),
    Description:  "PI network type",
},
```